src/views/download/download.scss

```diff
@@ -6,123 +6,171 @@
 }
 
 .download {
-    .title-banner {
-        &.masthead {
-            margin-bottom: 0;
-            background-color: $ui-blue-dark;
-            padding-bottom: 0;
+    .title {
+        margin-bottom: 1rem;
+        font-size: 2rem;
+    }
+
+    .centered {
```

Comment:
instead of naming this class after what it does, since it's only used in a few places, I suggest naming it after the type of component/tag that uses it. In this case, you might call it 'legacy-link'?

Reply:
yes! Done!